packages/angular_devkit/build_angular/package.json

```diff
@@ -69,7 +69,7 @@
     "terser": "5.3.1",
     "terser-webpack-plugin": "4.2.1",
     "tree-kill": "1.2.2",
-    "webpack": "4.44.1",
+    "webpack": "4.44.2",
```

Comment:
what happened, that this is not reflected in the release from two days ago?

![image](https://user-images.githubusercontent.com/905328/94269682-44c4eb80-ff3f-11ea-9223-495a2946c8c3.png)